tests/test_examples.py

```diff
@@ -0,0 +1,49 @@
+import pathlib
+import subprocess
+
+import pytest
+from pytest_notebook import execution, notebook
+
+EXAMPLES_DIR = pathlib.Path("examples/")
```

Comment:
I'm a bit nervous of using relative paths. We don't want to use e.g. `pkgresource` because `examples/` isn't installed. But we could use a path relative to the *source* file, which should work wherever you run `pytest` from?

https://stackoverflow.com/a/3718923/14507224 gives an example of this